Packs/Ironscales/IncidentFields/incidentfield-Ironscales-Classification.json

```
@@ -0,0 +1,29 @@
+{
+    "associatedToAll": false,
```

this should be:
```suggestion
    "associatedToAll": false,
    "associatedTypes": [
        "Ironscales"
    ],

```
*for all incident types*